tests/integration/targets/sysrc/main/main.yml

```diff
@@ -0,0 +1,223 @@
+---
+- name: Test on FreeBSD VMs
+  when:
+    - ansible_facts.virtualization_type != 'docker'
+    - ansible_facts.distribution == 'FreeBSD'
+  block:
+    ##
+    ## sysrc - /etc/rc.conf manipulation
+    ##
+    - name: add to /etc/rc.conf
+      sysrc:
+        name: sysrc_test
+        value: test
+      register: sysrc_test0
+
+    - name: get file content
+      shell: "cat /etc/rc.conf | egrep -v ^\\#"
+      register: sysrc_content0
+
+    - name: add to /etc/rc.conf
+      sysrc:
+        name: sysrc_test
+        value: test
+      register: sysrc_test0_change
+
+    - name: validate results
+      assert:
+        that:
+          - "sysrc_test0 is changed"
+          - "sysrc_test0_change is not changed"
+          - "'sysrc_test=\"test\"' in sysrc_content0.stdout_lines"
+
+    - name: Remove from /etc/rc.conf
+      sysrc:
+        name: sysrc_test
+        state: absent
+      register: sysrc_test1
+
+    - name: get file content
+      shell: "cat /etc/rc.conf | egrep -v ^\\#"
+      register: sysrc_content1
+
+    - name: Remove from /etc/rc.conf
+      sysrc:
+        name: sysrc_test
+        state: absent
+      register: sysrc_test1_change
+
+    - name: validate results for key removal
+      assert:
+        that:
+          - sysrc_test1 is changed
+          - sysrc_test1_change is not changed
+          - '"sysrc_test1" not in sysrc_content1.stdout_lines'
+          - '"sysrc_test11" not in sysrc_content1.stdout_lines'
```

Comment:
Why are you testing for this value?

Reply:
I don't remember the specifics of why I did that, but I've removed it.